presto-main/src/main/java/com/facebook/presto/operator/aggregation/fixedhistogram/FixedDoubleBreakdownHistogram.java

```diff
@@ -45,9 +45,9 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(FixedDoubleBreakdownHistogram.class).instanceSize();
 
-    private int bucketCount;
-    private double min;
-    private double max;
+    private final int bucketCount;
```

Comment:
either put the `final` changes in their own commit, or rename this commit to indicate that it's general "clean up"